data/hai/hai reveal 6 end.txt

```diff
@@ -50,21 +50,27 @@ mission "Hai Reveal [C01] Cleanup"
+			`	Apparently, the Syndicate is technically part of the Republic and awarded no special diplomatic status, so they've been busy with corporate matters elsewhere. Xilin finds you between proceedings and warns you, "Terry seems to have disappeared from Syndicate communications, though I won't hazard a guess as to their internal politics." A warning that carries no surprise at this point.`
+			`	It seems this station has seen serious renovations in the past weeks. You find yourself standing with the diplomats in a room now called the grand Borealis Chamber, "Where Mountains Touch The Sky," as Teeneep makes the first publicly broadcast speech from the Hai to humanity. The three heads of state announce a resolution granting safe passage to all Hai in human space, as long as they register with customs on Mountaintop before entering, and Teeneep expresses similar sentiments for humans visiting Hai space. It is announced that extensive surveillance fleets will be posted on both sides of the wormhole to ensure safety. This meeting will now turn into a multi-day summit to discuss the creation of a formal treaty.`
+			`	As you leave the summit for lunch, Xilin and Sayari intercept you and seem quite worried. `
 			choice
-				`	(Relax.)`
-			`Just  as you finish breakfast, when you think that maybe you'll be able to get on with your own life, there's a knock on the door of your station quarters. A slightly sheepish looking Sayari and Xilin are waiting for you when you open the door.`
-			`	"We need to talk."`
-			`	You feel the deepest of sighs settle into your bones, but at this point you just gesture them inside. It would have been too easy if it was all over, wouldn't it?`
-			`	"It seems that somewhere along the way, a shipload of kidnapped Hai didn't make it to a processing area. The Hai were left on Sunracer, and the ship appears to have disappeared," begins Xilin.`
+				`	"What's wrong?"`
+					goto need2talk
+				`	"Hey, don't worry. Whatever it is, I'm sure we've been through worse."`
+			`	Sayari rolls her eyes and says, "Maybe so, but there is one more minor disaster to be averted." You follow her to a private meeting room.`
+				goto whut
+			label need2talk
+			`	"We need to talk," says Xilin`
+			`	You feel the deepest of sighs settle into your bones, but at this point you just follow them to a private meeting room. It would have been too easy if it was all over, wouldn't it?`
+			label whut
```

Comment:
```suggestion
				`	"What's wrong?"`
					goto need2talk
				`	"Hey, don't worry. Whatever it is, I'm sure we've been through worse."`
			`	Sayari rolls her eyes and says, "Maybe so, but there is one more minor disaster to be averted." You follow her to a private meeting room.`
				goto whut
			label need2talk
			`	"We need to talk," says Xilin.`
			`	You feel the deepest of sighs settle into your bones, but at this point you just follow them to a private meeting room. It would have been too easy if it was all over, wouldn't it?`
			label whut
```
This could be rearranged, instead of telling the player what they feel. "We need to talk" could come first, then have the player ask what's wrong.

Reply:
I changed it to "Trouble. We need to talk,"